zio-http/src/main/scala/zio/http/model/Patch.scala

```diff
@@ -1,4 +1,6 @@
-package zio.http
+package zio.http.model
+
+import zio.http.Header
```

Comment:
Is this ok or should it be `import zio.http.model.Headers.Header`

Reply:
Not sure about Header if it must belong to model or not